src/allmydata/test/test_system.py

```diff
@@ -2504,15 +2486,11 @@ def _check_cp_out3(out_and_err):
         d.addCallback(run, "cp", files[5], "tahoe:file4")
         d.addCallback(run, "ls")
         d.addCallback(_check_ls, ["file3", "file3-copy", "file4"])
-        d.addCallback(run, "get", "tahoe:file4")
-        d.addCallback(_check_stdout_against, 5)
 
```

Comment:
I applaud the effort to simplify this test :)  .. but some of these "get" things look like they're not just "testing tahoe-get" but are confirming that some other command did the right thing (like here, confirming the "cp" actually copied the right content, I guess).

Reply:
I'm not sure where this PR stands .. I guess the whole point is that "get"s are tested elsewhere?

Reply:
I think your first comment was reasonable.  I haven't tried to go back and improve this.  I guess we could close this PR for now to reflect the fact that it's not actively being worked on?